bin/node/runtime/src/lib.rs

```diff
@@ -895,6 +895,22 @@ impl pallet_vesting::Trait for Runtime {
 	type WeightInfo = weights::pallet_vesting::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const AssetDepositBase: Balance = 100 * DOLLARS;
+	pub const AssetDepositPerZombie: Balance = 1 * DOLLARS;
+}
+
+impl pallet_assets::Trait for Runtime {
+	type Event = Event;
+	type Balance = u64;
```

Comment:
This is incompatible with the standard `type Balance = u128` defined in the standard Substrate node and node-template. This should use the `Balance` type to be compatible.

Reply:
Why?

Reply:
mmh, I guess I would expect a runtime to follow the convention of only having one type named `Balance` to avoid confusion. I see your point that they can be different in principle, though.

> `u128` is not a "standard"

maybe standard was the wrong word, I meant standard as in "default" or "configured" or "example"

Reply:
> `u128` is not a "standard" and there is no reason to think that fungible asset pallet balances should necessarily be the same type as a balance elsewhere in the chain.

@JesseAbram and I tried setting a Substrate chain up with `u64` as the assets `Balance` type, but polkadot-js assumes that `T::Balance` equals `Balance` (the globally defined type), so in practice we are not able to interact with the chain. (This is a similar issue to the other issue you already addressed.)

We'll get in touch with Jaco about improving polkadot-js on this, but until then we need to configure it to be the same concrete type or change the name.